app/controllers/api/v1/merchants/revenue_controller.rb

```diff
@@ -1,26 +1,23 @@
 class Api::V1::Merchants::RevenueController < ApplicationController
-  include ActionView::Helpers::NumberHelper  
```

Comment:
**Removed this because I implemented a serializer to format the revenue numbers** (see lines 12 and 21)